resources/assets/coffee/react/profile-page/beatmap-playcount.coffee

```diff
@@ -31,26 +34,36 @@ export class BeatmapPlaycount extends React.PureComponent
             a
               className: "#{bn}__title"
               href: beatmapUrl
-              "#{beatmapset.title} [#{beatmap.version}] "
+              "#{getTitle(beatmapset)} [#{beatmap.version}] "
               span
                 className: "#{bn}__title-artist"
-                osu.trans('users.show.extra.beatmaps.by_artist', artist: beatmapset.artist)
+                osu.trans('users.show.extra.beatmaps.by_artist', artist: getArtist(beatmapset))
           div
             className: "#{bn}__info-row u-ellipsis-overflow"
             span
               className: "#{bn}__artist"
-              dangerouslySetInnerHTML:
-                __html: osu.trans('users.show.extra.beatmaps.by_artist', artist: "<strong>#{_.escape(beatmapset.artist)}</strong>")
+              el StringWithComponent,
+                pattern: osu.trans 'users.show.extra.beatmaps.by_artist'
+                mappings:
+                  ':artist':
+                    strong
+                      key: 'artist'
+                      _.escape(getArtist(beatmapset))
             ' ' # separator for overflow tooltip
             span
               className: "#{bn}__mapper"
-              dangerouslySetInnerHTML:
-                __html: osu.trans 'beatmapsets.show.details.mapped_by',
-                  mapper: laroute.link_to_route 'users.show',
-                    beatmapset.creator
-                    { user: beatmapset.user_id }
-                    class: "#{bn}__mapper-link js-usercard"
-                    'data-user-id': beatmapset.user_id
+              el StringWithComponent,
+                pattern: osu.trans 'beatmapsets.show.details.mapped_by'
+                mappings:
+                  ':mapper':
+                    span
+                      className: "#{bn}__mapper-link"
+                      key: 'mapper'
+                      el UserLink,
```

Comment:
the color is wrong because direct styling (on `a`) has higher precedence than inherited (from `__mapper-link`)